### PR TITLE
updated to the latest webassembly js api

### PR DESCRIPTION
**Pull request by wanderer**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -44,8 +44,8 @@ module.exports = class Kernel {
   //       otherwise execution succeeded
   codeHandler (code, ethInterface = new Interface(new Environment())) {
     const debugInterface = new DebugInterface(ethInterface.environment)
-
-    const instance = Wasm.instantiateModule(code, {
+    const module = WebAessembly.Module(code)
```
Comment (axic) [inline index.js]: Typo here and below.

File: index.js [hunk 2]
```diff
@@ -44,8 +44,8 @@ module.exports = class Kernel {
   //       otherwise execution succeeded
   codeHandler (code, ethInterface = new Interface(new Environment())) {
     const debugInterface = new DebugInterface(ethInterface.environment)
-
-    const instance = Wasm.instantiateModule(code, {
```
Comment (axic) [inline index.js:48]: Can you make it work with both ways? i.e. check for the presence of `instantiateModule.`

File: index.js (hunk 1 above)
Comment (wanderer) [reply index.js]: fixed

File: index.js (hunk 2 above)
Comment (wanderer) [reply index.js:48]: I could but this is the incorrect API and unique to V8 for a breif time. Do you think that would be useful?

**Pull request merged**